content/en/docs/quick-start/all-in-one-on-linux.md

```diff
@@ -16,19 +16,37 @@ For those who are new to KubeSphere and looking for a quick way to discover the
 
 To get started with all-in-one installation, you only need to prepare one host according to the following requirements for hardware and operating system.
 
-### Hardware recommendations
-
-| System                                                 | Minimum Requirements                        |
-| ------------------------------------------------------ | ------------------------------------------- |
-| **Ubuntu** *16.04, 18.04*                              | CPU: 2 Cores, Memory: 4 G, Disk Space: 40 G |
-| **Debian** *Buster, Stretch*                           | CPU: 2 Cores, Memory: 4 G, Disk Space: 40 G |
-| **CentOS** *7*.x                                       | CPU: 2 Cores, Memory: 4 G, Disk Space: 40 G |
-| **Red Hat Enterprise Linux 7**                         | CPU: 2 Cores, Memory: 4 G, Disk Space: 40 G |
-| **SUSE Linux Enterprise Server 15/openSUSE Leap 15.2** | CPU: 2 Cores, Memory: 4 G, Disk Space: 40 G |
+### Hardware Recommendations
+
+<table>
```

Comment:
Why do we modify the table format to an HTML format?

Reply:
Because the markdown table does not support functions such as adjusting column width, or adding a bullet list. By using HTML, the format looks neat and beautiful. I wonder what is your opinion?

Reply:
It's wired to embed an HTML table. Maybe we could try things from  https://stackoverflow.com/questions/36121672/set-table-column-width-via-markdown, like style the table with css

Reply:
Styling each table with a CSS file makes things even more complicated. We use HTML tables because markdown tables do not even support the most basic features necessary for document writing, as mentioned by serenashe. I believe using HTML tables is a good choice unless there is a solid reason why HTML tables should not be used, in terms of user experience.